### DISCUSSION TITLE
文档写成这样....是不是生怕用的人看懂?

### DISCUSSION THREAD

**Discussion by k7cd** [alibaba/formily#4121] [Ideas]
如题...

**Comment by busy-dog** [comment]
三月没更新了

**Comment by xmsz-stu** [comment]
是的 老早以前看过 一脸懵 ，然后现在又回来看发现还是一样... 甚至感觉处理都慢了

**Comment by shubinla** [comment]
瞎说什么大实话 🤪

**Comment by John251314** [comment]
Mark

**Comment by John251314** [comment]
吐槽一下，表单设计器的代码以及github连接，指向的都是disignable中formily 1.0的代码，简直了

**Comment by dennis-leee** [comment]
主创似乎已经离职了，都是后来者在维护...

**Comment by AttackXiaoJinJin** [comment]
文档开头就和你说了上手门槛高🤣🤣

**Reply by moluoxixi** [reply]
这玩意这么复杂的json协议，真的有人用吗？为了接入耗费巨大，都不用考虑接自己的组件库就需要很久时间，文档也缺失

**Reply by baojianbj** [reply]
@moluoxixi 主要是文档缺失，没有详细说明，根本就是自己各种尝试
<img width="1418" height="143" alt="image" src="https://github.com/user-attachments/assets/7f781ebd-2abd-4a70-8003-621578ed7792" />
写的挺可笑的，如果文档详细的话，谁会问低级问题呢，就不考虑考虑是他们文档的问题？？？

**Reply by a550913664** [reply]
> **_简直就是一坨臭🐕💩，这里不是骂人哈，只是吐槽一下这个文档。_**

**Comment by a550913664** [comment]
`毋庸置疑`
_**简直就是极品文档好吧**_

**Comment by hikaruUwU** [comment]
拉完了，特别是表单设计器

**Comment by KunHuoJ** [comment]
主要是没人管了，文档一团乱麻
